source/common/http/http1/codec_impl.cc

```diff
@@ -436,14 +436,15 @@ void ConnectionImpl::dispatch(Buffer::Instance& data) {
 }
 
 size_t ConnectionImpl::dispatchSlice(const char* slice, size_t len) {
-  ssize_t rc = http_parser_execute(&parser_, &settings_, slice, len);
-  if (HTTP_PARSER_ERRNO(&parser_) != HPE_OK && HTTP_PARSER_ERRNO(&parser_) != HPE_PAUSED) {
+  const size_t bytes_read = parser_execute(&parser_, &settings_, slice, len);
+
+  if (parser_get_errno(&parser_) != HPE_OK && parser_get_errno(&parser_) != HPE_PAUSED) {
     sendProtocolError();
     throw CodecProtocolException("http/1.1 protocol error: " +
-                                 std::string(http_errno_name(HTTP_PARSER_ERRNO(&parser_))));
+                                 std::string(parser_errno_name(parser_get_errno(&parser_))));
```

Comment:
There is a new opportunity for detailed errors here. One could potentially use `llhttp_get_error_reason` to get the detailed reason for the error.